src/collective/cover/layout.py

```diff
@@ -142,6 +144,22 @@ def accepted_ct_for_tile(self, tile_type):
 
         return json.dumps(accepted_ct)
 
+    def content_type_for_tile(self, tile_type, tile_id):
```

Comment:
I know you don't like `get_` and `set_`, but is perfectly fine to use them and names get better: `get_content_portal_type` and `get_content_uuid` for instance